build.gradle

```diff
@@ -71,19 +71,19 @@ configurations {
 }
 
 dependencies {
-    ktlint "com.github.shyiko:ktlint:0.29.0"
+    ktlint "com.pinterest:ktlint:0.34.2"
```

Comment:
Just curious, the version `0.34.2` was released in 2019, and do we plan to use a newer version?